smartcontract/service/wasmvm/runtime.go

```diff
@@ -513,125 +535,135 @@ func NewHostModule() *wasm.Module {
 			Host: reflect.ValueOf(RaiseException),
 			Body: &wasm.FunctionBody{}, // create a dummy wasm body (the actual value will be taken from Host.)
 		},
+		{ //23
+			Sig:  &m.Types.Entries[11],
+			Host: reflect.ValueOf(Sha256),
+			Body: &wasm.FunctionBody{}, // create a dummy wasm body (the actual value will be taken from Host.)
+		},
 	}
 
 	m.Export = &wasm.SectionExports{
 		Entries: map[string]wasm.ExportEntry{
-			"timestamp": {
-				FieldStr: "timestamp",
+			"ontio_timestamp": {
+				FieldStr: "ontio_timestamp",
 				Kind:     wasm.ExternalFunction,
 				Index:    0,
 			},
-			"block_height": {
-				FieldStr: "block_height",
+			"ontio_block_height": {
+				FieldStr: "ontio_block_height",
 				Kind:     wasm.ExternalFunction,
 				Index:    1,
 			},
-			"input_length": {
-				FieldStr: "input_length",
+			"ontio_input_length": {
+				FieldStr: "ontio_input_length",
 				Kind:     wasm.ExternalFunction,
 				Index:    2,
 			},
-			"call_output_length": {
-				FieldStr: "call_output_length",
+			"ontio_call_output_length": {
+				FieldStr: "ontio_call_output_length",
 				Kind:     wasm.ExternalFunction,
 				Index:    3,
 			},
-			"self_address": {
-				FieldStr: "self_address",
+			"ontio_self_address": {
+				FieldStr: "ontio_self_address",
 				Kind:     wasm.ExternalFunction,
 				Index:    4,
 			},
-			"caller_address": {
-				FieldStr: "caller_address",
+			"ontio_caller_address": {
+				FieldStr: "ontio_caller_address",
 				Kind:     wasm.ExternalFunction,
 				Index:    5,
 			},
-			"entry_address": {
-				FieldStr: "entry_address",
+			"ontio_entry_address": {
+				FieldStr: "ontio_entry_address",
 				Kind:     wasm.ExternalFunction,
 				Index:    6,
 			},
-			"get_input": {
-				FieldStr: "get_input",
+			"ontio_get_input": {
+				FieldStr: "ontio_get_input",
 				Kind:     wasm.ExternalFunction,
 				Index:    7,
 			},
-			"get_call_output": {
-				FieldStr: "get_call_output",
+			"ontio_get_call_output": {
+				FieldStr: "ontio_get_call_output",
 				Kind:     wasm.ExternalFunction,
 				Index:    8,
 			},
-			"check_witness": {
-				FieldStr: "check_witness",
+			"ontio_check_witness": {
+				FieldStr: "ontio_check_witness",
 				Kind:     wasm.ExternalFunction,
 				Index:    9,
 			},
-			"current_blockhash": {
-				FieldStr: "current_blockhash",
+			"ontio_current_blockhash": {
+				FieldStr: "ontio_current_blockhash",
 				Kind:     wasm.ExternalFunction,
 				Index:    10,
 			},
-			"current_txhash": {
-				FieldStr: "current_txhash",
+			"ontio_current_txhash": {
+				FieldStr: "ontio_current_txhash",
 				Kind:     wasm.ExternalFunction,
 				Index:    11,
 			},
-			"ret": {
-				FieldStr: "ret",
+			"ontio_ret": {
```

Comment:
before we had to use `ret` instead `return` since `return` is rust/c++ keyword. but now we can rename to `ontio_return` to make it more clear.